command-line.txt

```diff
@@ -1,11 +1,11 @@
 What is the command that displays your current working directory?
-your_answer = ____________
+your_answer = pwd
 
 What is the command to list all of the files and directories in your current working directory?
-your_answer = ____________
+your_answer = ls
 
 What is the command to move the image of the dirty puppy into the bath tub?
-your_answer = _____________
+your_answer =  mv dirty_puppy.rb/command-line-proficiency-2015 (biggest directory)/bath-tub-for-dirty-puppy
```

Comment:
this is close. I imagine that it would be more clear if the bathtub directory was in the repository already. see updated repo.